redbot/cogs/trivia/data/lists/2020.yaml

```diff
@@ -0,0 +1,48 @@
+AUTHOR: OofChair#0001
+When did Prince Harry and Meghan Markle leave the royal family?:
+- January
+- january
+When did COVID-19 first appear?:
+- December 2019
+- december 2019
+When did Kobe Bryant and Gianna Bryant die? (rest in peace):
+- January
+- january
+Which movie won the Oscar award for Best Film?:
+- Parasite
+- parasite
+What was the name of the streaming service that launched in mid-2020 but got shut down 6 months later?:
+- Quibi
+- quibi
+Where did scientists find evidence of water?:
+- Moon
+- moon
+- The moon
+- The Moon
+What popular application did President Trump threaten to ban from the US?:
+- tiktok
+- TikTok
+When did the Beirut explosion happen?:
+- August
+- august
+When was Kamala Harris, the first black female vice president, announced as Joe Biden's running mate?:
+- august
+- August
+When did WHO, or the World Health Organization, announce a public health emergency about COVID?:
+- january
+- January
+When did the worldwide cases of COVID-19 pass 10 million?:
+- june
+- June
+What streaming service had their subscriber count doubled during the first 3 months of 2020?:
+- Netflix
```

Comment:
You've added `Netflix` 2 times, one here and below this one, maybe remove one of them? since they does not look different :P

Reply:
Thanks MAX, but I think I'm gonna close this because I have no time to work on this currently because of life things, and I might reopen later.